bundles/org.openhab.binding.tibber/src/main/java/org/openhab/binding/tibber/internal/handler/TibberHandler.java

```diff
@@ -267,6 +275,36 @@ public void getURLInput(String url) throws IOException {
         }
     }
 
+    /**
+     * Builds the {@link TimeSeries} that represents the future tibber prices.
+     *
+     * @param today The prices for today
+     * @param tomorrow The prices for tomorrow.
+     * @return The {@link TimeSeries} with future values.
+     */
+    private TimeSeries buildTimeSeries(JsonArray today, JsonArray tomorrow) {
+        final TimeSeries timeSeries = new TimeSeries(TimeSeries.Policy.REPLACE);
+        mapTimeSeriesEntries(today, timeSeries);
+        mapTimeSeriesEntries(tomorrow, timeSeries);
+        return timeSeries;
+    }
+
+    private static void mapTimeSeriesEntries(JsonArray today, TimeSeries timeSeries) {
```

Comment:
```suggestion
    private static void mapTimeSeriesEntries(JsonArray prices, TimeSeries timeSeries) {
```

(or similar, since "today" is now misleading)

Reply:
done.